gsplat/rendering.py

```diff
@@ -991,3 +994,374 @@ def _getProjectionMatrix(znear, zfar, fovX, fovY, device="cuda"):
         render_colors.append(render_colors_)
     render_colors = torch.stack(render_colors, dim=0)
     return render_colors, None, {}
+
+
+###### 2DGS ######
+def rasterization_2dgs(
+    means: Tensor,
+    quats: Tensor,
+    scales: Tensor,
+    opacities: Tensor,
+    colors: Tensor,
+    viewmats: Tensor,
+    Ks: Tensor,
+    width: int,
+    height: int,
+    near_plane: float = 0.01,
+    far_plane: float = 1e10,
+    radius_clip: float = 0.0,
+    eps2d: float = 0.3,
+    sh_degree: Optional[int] = None,
+    packed: bool = True,
+    tile_size: int = 16,
+    backgrounds: Optional[Tensor] = None,
+    render_mode: Literal["RGB", "D", "ED", "RGB+D", "RGB+ED"] = "RGB",
+    sparse_grad: bool = False,
+    absgrad: bool = False,
+    rasterize_mode: Literal["classic", "antialiased"] = "classic",
+    distloss: bool = False,
+) -> Tuple[Tensor, Tensor, Dict]:
```

Comment:
You should modify the `Tuple[Tensor, Tensor, Dict]` to corresponding the what you return actually